C-Plus-Plus/math/Primefactorization.cpp

```diff
@@ -0,0 +1,48 @@
+//CPP program to prime factorize a number using sieve
+#include<iostream>
```

Comment:
Use bits/stdc++.h instead of multiple header files.

Reply:
Done @atarax665 .